docs/adr/0004-editing-microcopy.md

```diff
@@ -1,6 +1,6 @@
 # 4. Editing Microcopy
 
-Date: 2018-09-11
+Date: 2020-01-21
```

Comment:
Sorry to be splitting hairs. 

It'd be more accurate to flag this as:

```
Date: 2018-09-11    
Revised: 2020-01-21
```

Reply:
Sure. Done 👍